packages/eslint-plugin-pf-codemods/test/rules/refactor-breakpointmods.js

```diff
@@ -0,0 +1,99 @@
+const ruleTester = require('./ruletester');
+const rule = require('../../lib/rules/refactor-breakpointmods');
+
+ruleTester.run("refactor-breakpointmods", rule, {
+  valid: [
+    {
+      code: `<Flex alignSelf={{default: "alignSelfStretch"}}>
+        <FlexItem alignSelf={{default: "alignSelfStretch"}}></FlexItem>
+      </Flex>`
+    },
+    {
+      code: `<PageSection padding={{ default: 'noPadding' }}></PageSection>`
+    },
+    {
+      code: `<ToolbarItem visibility={{
+        default: 'visible',
+        lg: 'hidden'
+      }}></ToolbarItem>`
+    },
+    {
+      code: `<ToolbarToggleGroup spacer={{default: "spacerNone"}} show={{xl: 'show'}}></ToolbarToggleGroup>`
+    }
+  ],
+  invalid: [
+    {
+      code: `import { Flex, FlexItem, FlexModifiers } from '@patternfly/react-core';
+        <Flex breakpointMods={[
+          {modifier: FlexModifiers.column},
+          {modifier: FlexModifiers["align-self-stretch"]},
+          {modifier: 'spacer-none'},
+          {modifier: 'row', breakpoint: 'sm'}
+        ]}>
+          <FlexItem breakpointMods={[
+            {breakpoint: 'xl', modifier: FlexModifiers.column},
+            {modifier: FlexModifiers["align-self-stretch"], breakpoint: 'md'},
+            {modifier: FlexModifiers.row}
+          ]}></FlexItem>
+        </Flex>`,
+      output: `import { Flex, FlexItem, FlexModifiers } from '@patternfly/react-core';
```

Comment:
We also need to remove the imports `'FlexModifiers', 'FlexBreakpoints', /\w+BreakpointMod/`

Reply:
Updated `remove-unused-imports` rule to remove `FlexModifiers, FlexBreakpoints, FlexBreakpointMod, FlexItemBreakpointMod`.  There is still a `formatBreakpointMods` method used in react-core that I did not want to mistakenly remove using a broader regex match.